src/components/home/modals/Send.vue

```diff
@@ -634,6 +659,20 @@ export default {
                 } else {
                     this.coldPageId++
                 }
+                if (this.from3rdParty) {
+                    if (this.paymentRedirect.hasOwnProperty('callback')) {
+                        let url = this.paymentRedirect['callback']
+                        if (url.indexOf('?') !== -1) {
+                            url += '&tx_id=' + response.id
+                        } else {
+                            url += '?tx_id=' + response.id
+                        }
+                        this.$http.post(url).then(response => {
```

Comment:
If failed, the callback should retry later until 5 times. Suggest use `setTimeout` or `setInterval` to implement.